### PR TITLE
Implementation of linked list and various syntax fixes

### PR DESCRIPTION
**Pull request by vminin-solvd**
Changed system prints to logger.info and implemented linkedlist. Now will add collections.

### CODE REVIEW

**Comment by vminin-solvd** [comment]
I have now added all the collections I have branched for the next task to get ahead. Let me know of any issues.

File: zoo/linkedlist/CustomLinkedList.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    private static final Logger LOGGER = LogManager.getLogger(CustomLinkedList.class);
+    private CustomNode<T> head;
+
+    public CustomLinkedList() {
+
+        this.head = null;
+
+    }
+
+    public void addNode(CustomNode<T> node) {
```
Comment (ankochergin) [inline zoo/linkedlist/CustomLinkedList.java:17]: could you please add methods add(T), remove(T), getAll, get(index)?
all manipulations with nodes should be inside of this class

File: zoo/linkedlist/CustomLinkedList.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            if (node.next == null) {
+
+                this.head = null;
+            }
+
+            else {
+
+                this.head.next = node.next;
+            }
+        }
```
Comment (ankochergin) [inline zoo/linkedlist/CustomLinkedList.java:54]: please remove unnecessary empty lines is the code, here is an example: 
```suggestion
    public void removeNode(CustomNode<T> node) {
        CustomNode<T> previousNode = null;
        CustomNode<T> currentNode = this.head;
        CustomNode<T> nextNode = null;
        if (this.head == node) {
            if (node.next == null) {
                this.head = null;
            } else {
                this.head.next = node.next;
            }
        }
```

File: zoo/Main.java [hunk 3]
```diff
@@ -19,8 +21,18 @@ public class Main {
 
     private static final Logger LOGGER = LogManager.getLogger(Main.class);
 
-    public static void main(String[] args) throws nameException, LocationException, ExpiredTicketException, FeedAnimalException {
-
+    public static void main(String[] args) throws InvalidNameException, LocationException, ExpiredTicketException, FeedAnimalException {
+
+        CustomLinkedList linkedList = new CustomLinkedList();
```
Comment (ankochergin) [inline zoo/Main.java:26]: try to use your customLinkedList for some objects from your hierarchy, for example, store animals in it

**Pull request merged**